sdk/src/types/api/plugins/participation/types.rs

```diff
@@ -301,3 +301,32 @@ impl Participations {
         Ok(Self { participations })
     }
 }
+
+impl ParticipationEventData {
+    #[cfg(test)]
```

Comment:
Maybe move these tags up?

Reply:
sorry for the late reply ... 

hmm ... I had it on the `impl` block before, but we'ld have to move it back in if we ever decide to add a regular method to this type. If you think it's better up there then I am fine with moving it back up.

Reply:
I do think it's better at the top level. It's not a big deal to add more impl blocks.

Reply:
Done